pkg/report/table/table.go

```diff
@@ -63,10 +63,6 @@ func (tw Writer) Write(report types.Report) error {
 }
 
 func (tw Writer) write(result types.Result) {
-	if result.IsEmpty() {
```

Comment:
Doesn't this kind of change work?

`if result.IsEmpty() && result.Class != types.ClassVulnOSPkgs {`

Reply:
no, `Headers` will be shown without results

Reply:
I see. I think we can show an empty table for vulnerabilities in OS packages. It might be too much to display empty tables for language-specific packages.

Reply:
i thought that an empty table isn't nice.
also we already have additional checks for `secret` and `misconfiguration` scans:
https://github.com/aquasecurity/trivy/blob/6138f57bf98e92f389a349fe88291cc89af48cc3/pkg/report/table/secret.go#L43-L46

https://github.com/aquasecurity/trivy/blob/6138f57bf98e92f389a349fe88291cc89af48cc3/pkg/report/table/misconfig.go#L54-L57

but i can add a new condition to `if` statement, if you think it's better.

Reply:
>i thought that an empty table isn't nice.

OK. Could you share your suggestion with me?

Reply:
updated

Reply:
> https://github.com/aquasecurity/trivy/blob/6138f57bf98e92f389a349fe88291cc89af48cc3/pkg/report/table/secret.go#L43-L46
> 
> https://github.com/aquasecurity/trivy/blob/6138f57bf98e92f389a349fe88291cc89af48cc3/pkg/report/table/misconfig.go#L54-L57

if these changes are ok, let's remove unused checks, right?

Reply:
Sounds good.